landoapi/spec/swagger.yml

```diff
@@ -186,6 +186,41 @@ paths:
           schema:
             allOf:
               - $ref: '#/definitions/Error'
+  /requestSecApproval:
+    post:
+      operationId: landoapi.api.revisions.request_sec_approval
+      description: |
+        Submit a sanitized, safe-to-land commit message for a security-sensitive
+        revision. This starts the Security Bug Approval Process.
+      parameters:
+        - name: data
+          description: The data we want to submit
+          required: true
+          in: body
+          schema:
+            type: object
+            required:
+              - revision_phid
+              - sanitized_message
+            properties:
+              revision_phid:
```

Comment:
We accept revision IDs everywhere else, converting to PHIDs as needed behind the scenes. Probably best to keep this consistent. Sorry I didn't catch this on previous passes.